ONSdigital/response-operations-ui
# RAS-873: Show supplementary dataset ID in Rops collection exercise

# What and why?
Adds a panel to notify a Response Operations user that this collection exercise has pre populated data linked with it.

# How to test?
- Deploy this PR to your environment
- Ensure that https://github.com/ONSdigital/rm-collection-exercise-service/pull/318 is either merged or deploy it to your environment
- Set up a collection exercise in Response Operations (only period ID is needed)
- Place a pubsub message using and replacing your namespace, survey ID and period ID:
```
gcloud pubsub topics publish projects/ras-rm-dev/topics/supplementary-data-service-[your namespace] --message='{
    "survey_id": [your survey ID],
    "period_id": [your period ID],
    "form_types": [
       "0017",
       "0123",
       "0001"
      ],
    "title": "A dataset version 4 for survey 2012 period 2013",
    "sds_published_at": "2023-07-17T14:46:36Z",
    "total_reporting_units": 2,
    "schema_version": "v1.0.0",
    "sds_dataset_version": 4,
    "filename": "373d9a77-2ee5-4c1f-a6dd-8d07b0ea9793.json",
    "dataset_id": "b9a87999-fcc0-4085-979f-06390fb5dddd"
}'
```
- Refresh the collection exercise in Response Operations, now a panel saying `Pre-Populated data is available for this sample` should appear at the top (in line with the design https://www.figma.com/file/uUb6GDGbgYVILiK5wP9wqi/CE-Reporting-period-page?type=design&node-id=2220-73919&mode=design)


# Trello
[RAS-873](https://jira.ons.gov.uk/browse/RAS-873)
